client/src/app/components/Header/header.css

```diff
@@ -0,0 +1,122 @@
+h1 {
+  font-size: 20px;
+  margin: 10px;
+  color: #312a31;
+}
+
+.header {
+  width: 100%;
+  margin: 0;
+  padding: 5px;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  border-bottom: 1px solid #e8e9eb;
+  box-shadow: 0 2px 2px 2px #e8e9eb;
+  font-size: 16px;
+}
+
+.logo {
+  font-size: 25px;
+  color: #ec5252;
+}
+
+.nav-container {
+  display: flex;
+  align-items: center;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+}
+
+li {
+  margin: 0 5px 0 5px;
+  padding: 10px;
+  border: 1px solid transparent;
+}
+
+li:hover {
+  background: #f2f3f5;
+  border: 1px solid #e8e9eb;
+  cursor: pointer;
+}
+
+.input-container {
+  width: 40vw;
+  height: 35px;
+  display: flex;
+  align-items: center;
+  border: 1px solid #e8e9eb;
+  display: flex;
+  justify-content: space-between;
+}
+.searchBarGrey {
```

Comment:
The class name should not reveal implementation details such as the color (gray). This might change in the future. Instead it should reflect the state (active, focused, etc)